layers/+spacemacs/spacemacs-evil/funcs.el

```diff
@@ -61,7 +61,6 @@ Otherwise, revert to the default behavior (i.e. enable `evil-insert-state')."
       (evil-escape-mode t)
     (evil-escape-mode -1)))
 
-
```

Comment:
This is a form-feed character, it's used in several places, for example in the Spacemacs buffer, and even 2 other times in this same file `spacemacs-evil/funcs.el`.

They just look like empty lines in the github diff, but they look like this in Spacemacs:

![emacs_2019-04-24_16-44-28](https://user-images.githubusercontent.com/13420573/56669013-c82a3f80-66b0-11e9-9cfc-bce3b3728b77.png)

In other words, they should not be removed.

Reply:
Yes, I know it.
But in this file, all the other form-feed characters are placed right before a comment, and I think it means start of a new section. But this one doesn't like this, so I wonder whether we should delete it.

Reply:
You might be right, it seems logical to only add one before a comment/section. So it's probably fine to remove this one.